### DISCUSSION TITLE
Add "note" or color highlight overlay option to items in bank

### DISCUSSION THREAD

**Discussion by SassLife** [runelite/runelite#15891] [Q&A]
This would be super helpful for your people with bad memory! Sometimes I forget why I keep certain items.. !

**Comment by Nightfirecat** [comment]
You can already add tags to bank items via the Bank Tags plugin, which could be used for this purpose.

**Reply by SassLife** [reply]
so I mean like a highlight option maybe. you know inventory tags how it can make it a color in the inventory... it would be so nice to be able to highlight certain items in my bank so I stop selling them because I end up getting the drop again from raids or whatever.

**Reply by SassLife** [reply]
So I play ba a lot, and I would stop selling the drops if I could remember what they were. And like impling jars when I open them.. I would stack them up and not sell the loot if I could remember what it was. So like a color overlay that you could add to items in the bank would be sooooo nice
